src/DI/ServiceLookup/ConstantCallSite.cs

```diff
@@ -16,5 +16,6 @@ public ConstantCallSite(Type serviceType, object defaultValue)
 
         public Type ServiceType => DefaultValue.GetType();
         public Type ImplementationType => DefaultValue.GetType();
+        public CallSiteKind Kind { get; } = CallSiteKind.Constant;
```

Comment:
public CallSiteKind Kind => CallSiteKind.Constant;

Reply:
So JIT can do it's JITty stuff